README.md

```diff
@@ -0,0 +1,248 @@
+<a name="readme-top"></a>
+
+<!--
+
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+ -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello microverse from omar´S desing <a name="about-project"></a>
+
+
+**helolo microverse** is an initial proyect where im learning how to use basic CSS. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Github</summary>
+  <ul>
+    <li><a href="">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Git</summary>
+  <ul>
+    <li><a href="">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Visaul Code</summary>
+  <ul>
+    <li><a href="">PostgreSQL</a></li>
+  </ul>
```

Comment:
[OPTIONAL] - Please let's update the tech stack by removing `React.js`, `Express.js`, and `PostgresSQL` as these are out of the scope of this particular project 😀. then we can include the technologies that we have used in the project. 💪